### PR TITLE
Added Buttons

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added Primary , Secondary , Icon , Link and Floating Buttons

### CODE REVIEW

**Comment by harshalini** [comment]
Use rem instead of px

File: components/buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+/* Floating Button */
+
+.floating-btn{
+    color: white;
+    font-size: 20px;
+    background-color: var(--floating-btn);
+    font-weight: bolder;
+    padding: 10px 13px;
```
Comment (JantuDeb) [inline components/buttons/buttons.css:51]: I would say use same padding for all the buttons, if this necessary for floating button then it's fine.

**Review by JantuDeb — APPROVED** [review]
This is looking good you can merge.

File: components/buttons/buttons.css (hunk 1 above)
Comment (atharvabhanage02) [reply components/buttons/buttons.css:51]: Yeah ! Thanks for suggesting that !

**Comment by atharvabhanage02** [comment]
> Use rem instead of px

Use rem instead of px
I have tried my best to use rem's wherever possible , but I'll try to keep the units in rem's
Thanks for suggestion.

**Pull request merged**